### DISCUSSION TITLE
Deletion of stock which are not updated

### DISCUSSION THREAD

**Discussion by cotton1234** [BennyThadikaran/eod2#160] [General]
Hello Benny,

What is the criteria of deletion of stock file which are not updated.

Example - SABTN

This is not updated since Jan.

Thanks
Saurabh

**Comment by BennyThadikaran** [comment]
If a stock is not updated for more than 365 days it is deleted. I kept it this way to avoid deleting stocks which are in temporary suspension.

**Reply by cotton1234** [reply]
> If a stock is not updated for more than 365 days it is deleted. I kept it this way to avoid deleting stocks which are in temporary suspension.
> […](#)
> On 11 July 2024 15:40:57 UTC, Saurabh ***@***.***> wrote: Hello Benny, What is the criteria of deletion of stock file which are not updated. Example - SABTN This is not updated since Jan. Thanks Saurabh -- Reply to this email directly or view it on GitHub: #160 You are receiving this because you are subscribed to this thread. Message ID: ***@***.***>

Fine Benny.
